### PR TITLE
fix(medic): delete mixed and non mixes old benchmarks

### PR DESCRIPTION
**Pull request by megglos**
## Description

Extends the medic script to delete mixed and non mixed benchmarks.

### CODE REVIEW

**Comment by megglos** [comment]
I tried it and it deleted the CW 35 benchmarks. Didn't push the commits though as it was a test, will do that manually.

File: createMedicCWBenchmark.sh [hunk 1]
```diff
@@ -5,11 +5,14 @@ set -euxo pipefail
 cw=$(date +%V)
 if [ $cw -gt 4 ]
 then
-  nameOfOldestBenchmark=$(ls | grep medic-cw- | sort | head -n 1)
-  ./deleteBenchmark.sh $nameOfOldestBenchmark
+  cw_to_delete=$((cw-4))
```
Comment (Zelldon) [inline createMedicCWBenchmark.sh:8]: Cool it seem to work :)

**Review by ChrisKujawa — APPROVED** [review]
Thanks :100:

**Review by remcowesterhoud — APPROVED** [review]
LGTM! 🚀

**Pull request merged**